reuse/contribute-to-docs.txt

```diff
@@ -28,18 +28,101 @@ Use the :guilabel:`Give feedback` button at the top of any page to create a GitH
 New content
 -----------
 
-While contributing new content, it is easier to work with the docs on your local machine. You can submit a PR after all the checks have passed and things looks satisfactory. You'll need ``make`` and ``python3`` installed on your system.
+When adding new content, it's easier to work with the documentation on your local machine. Once you've made your changes, ensure all checks have passed and everything looks satisfactory before submitting a pull request (PR). To run the checks, you must have ``make`` and ``python3`` installed on your system.
+
+Use the `Diátaxis`_ approach to create new categories for this documentation. These categories can have subcategories within them as well. The documentation has a base ``index.rst`` file, which is usually the landing page. The ``index.rst`` file contains the documentation title, content introducing the documentation and its purpose, relevant links, and a toctree. The toctree lists the categories contained in the documentation in the order they appear in the navigation. Each category and subcategory also contains an ``index.rst`` file similar to the documentation's base ``index.rst`` file.
+
```

Comment:
This whole portion along with the tree can go under the "Create content" subsection. Also, if possible let's try to make this explanation simpler. Something like:
"Choose the appropriate folder for your content. The folders within each project are mapped to the `Diátaxis`_  categories of tutorial, how-to guides, explanation and reference. If required, the categories can have subcategories as well, as shown in the tree structure below. Also, each folder includes an ``index.rst`` file, which acts as a landing page for the folder. 
..code::  
( tree )
If your required category or subcategory is absent, create them using the instructions given below. Then add your content by creating a new page."

Also in the tree, can we call the top-level 'project' instead of 'documentation' since we use that terminology in the other sections. And could you show the sample subcategories under how-to guides instead of tutorials? This is because usually a product has very few tutorials and won't need subcategories. Moreover, this documentation set has sub-categories only in how-to guides.